src/main/java/it/gov/pagopa/rtd/ms/rtdmsdecrypter/service/DecrypterImpl.java

```diff
@@ -79,17 +79,20 @@ public BlobApplicationAware decrypt(BlobApplicationAware blob) {
       blob.setStatus(BlobApplicationAware.Status.DECRYPTED);
       log.info("Blob {} decrypted.", blob.getBlob());
 
-    } catch (Exception ex) {
-      // Should throw an IOException just like decryptFile, this creates problems in the event
-      // handler where the method chaining doesn't allow Exception throws
-      log.error("Cannot decrypt {}: {}", blob.getBlob(), ex.getMessage());
+    } catch (IOException e) {
+      log.warn("{}: {}", e.getMessage(), blob.getBlob());
```

Comment:
Probably minor issue but still: it's not clear to me why this log severity has passed from error to warn. I understand that the IOException is thrown when the file size is zero, but there are many other case that can raise that particular exception. What's the rationale behind this particular change?

Reply:
You are right.
I wanted to log a warning, and not an error, in case of empty encrypted file.
But i overlooked other IOException causes.
Maybe I can throw a ParseException and log a warning if the encrypted file is empty.
What do you think?

Reply:
Good idea: throw another specific exception for the 'empty file' case, and leave the IOException to catch a broader set of unexpected errors.